.env.local

```
@@ -2,10 +2,28 @@ ROOT_LOGGING_LEVEL=info
 APP_LOGGING_LEVEL=info
 WEB_LOGGING_LEVEL=info
 
+MONGO_HOST="localhost"
+MONGO_PORT=28017
+MONGO_USERNAME=admin
+MONGO_PASSWORD=password
+MONGO_SSL_ENABLED=false
+MONGO_DB_NAME=wallet
+MONGO_MIN_POOL_SIZE=0
+MONGO_MAX_POOL_SIZE=20
+MONGO_MAX_IDLE_TIMEOUT_MS=60000
+MONGO_CONNECTION_TIMEOUT_MS=1000
+MONGO_SOCKET_TIMEOUT_MS=10000
+MONGO_SERVER_SELECTION_TIMEOUT_MS=2000
+MONGO_WAITING_QUEUE_MS=2000
+MONGO_HEARTBEAT_FREQUENCY_MS=5000
+
 SCHEDULER_CDC_QUEUE_NAME=pagopa-wallet-cdc-queue
 SCHEDULER_CDC_QUEUE_TTL_SECONDS=7200
 SCHEDULER_CDC_QUEUE_CONNECTION_STRING="DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;QueueEndpoint=http://localhost:10001/devstoreaccount1"
 SCHEDULER_CDC_QUEUE_VISIBILITY_TIMEOUT_SECONDS=0
 
 SCHEDULER_CDC_SEND_RETRY_MAX_ATTEMPTS=1
-SCHEDULER_CDC_SEND_RETRY_INTERVAL_IN_MS=1000
\ No newline at end of file
+SCHEDULER_CDC_SEND_RETRY_INTERVAL_IN_MS=1000
+
+WALLET_SEARCH_STATUS=CREATED
```

add those variables to dev and uat helm values too.
The searched wallet status should be validated, right?
```suggestion
WALLET_SEARCH_STATUS=VALIDATED
```